Parsers/ASimNetworkSession/Parsers/ASimNetworkSession.yaml

```diff
@@ -1,7 +1,7 @@
 Parser:
   Title: Network Session ASIM parser
-  Version: '0.7'
-  LastUpdated: Jan 17, 2024
+  Version: '0.7.2'
+  LastUpdated: June 5, 2024

```

Comment:
"LastUpdated" property exists but is not correct format. The expected format is, for example, "Jun 29, 2024"

Reply:
Done